Bitly/Bitly_Get_a_Clicks_Summary_for_a_Bitlink.ipynb

```diff
@@ -0,0 +1,241 @@
+   "id": "4253af9d-ad21-42da-b342-ed07dc870ab2",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91a8f1fe-89b0-443e-8129-5993b550dd5c",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d76e832-ce05-4e4d-ba60-dcb30f584989",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "import requests",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc385dae-5c6a-468b-a3c8-f77557762ad6",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup Variables\n- `token`: [Bitly Authentication](https://dev.bitly.com/authentication.html)\n- `bitlink`: Bitlink to get the Clicks Summary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8951ce10-e6de-4119-b47e-76d60652841d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "token = \"<YOUR_TOKEN>\"\nbitlink = \"<YOUR_BITLINK>\"",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a4c6817-1f01-43fb-a2af-7b19f67df884",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a68b8623-20f6-417f-9440-ee87d005aff2",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Get Clicks Summary for a Bitlink"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b6231ab-932f-4264-b912-977c47256b33",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "This function will get a Clicks Summary for a Bitlink using the Bitly API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e959b46c-6357-404a-825a-aa8b40fa038a",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "def get_clicks_summary_for_bitlink(token, bitlink):\n    url = f\"https://api-ssl.bitly.com/v4/bitlinks/{bitlink}/clicks/summary\"\n    headers = {\"Authorization\": f\"Bearer {token}\"}\n    response = requests.get(url, headers=headers)\n    return response.json()",
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "830ac5bd-4557-479a-87a2-18d3d3ab618a",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a92c4ecf-bdc3-48b3-b9da-74a7d25e5db8",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Display result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9599ccd-23d7-4488-837c-95af632141ce",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": "clicks_summary = get_clicks_summary_for_bitlink(token, bitlink)\nclicks_summary",
```

Comment:
Getting a  minor: JSONDecodeError

<img width="1328" alt="Screenshot 2023-02-24 at 7 46 38 PM" src="https://user-images.githubusercontent.com/81156510/221200795-7e8e58cf-e3f9-4e04-b707-4144cfa7192a.png">

Reply:
@srini047, do you try this code with one of your bitlink and setup your token? It's working on my side

Reply:
@FlorentLvr Yes, I did try with my token & one of the bitly links that I have. I tried again. The same result.

Reply:
@srini047, let's check together on Slack! I can't reproduce it, it's working on my side.

Reply:
Sure @FlorentLvr